package.json

```
@@ -32,7 +32,7 @@
     "extend": "^2.0.0",
     "he": "^1.1.1",
     "iconv": "^2.3.4",
-    "irc": "github:matrix-org/node-irc#9028c2197c216dd8e6fc2cb3cc07ce2d6bf741a7",
+    "irc": "git+https://github.com:matrix-org/node-irc.git#9028c2197c216dd8e6fc2cb3cc07ce2d6bf741a7",
```

This should force HTTPS
```suggestion
    "irc": "git+https://matrix-org@github.com/matrix-org/node-irc.git#9028c2197c216dd8e6fc2cb3cc07ce2d6bf741a7",
```